PowerValidatedSolutions.psm1

```diff
@@ -19611,6 +19611,78 @@ Function Get-NsxtLdap {
 }
 Export-ModuleMember -Function Get-NsxtLdap
 
+Function New-NsxtLdap {
+    <#
+        .SYNOPSIS
+        Add an LDAP identity source
+
+        .DESCRIPTION
+        The New-NsxtLdap cmdlet adds an LDAP identity source
+
+        .EXAMPLE
+        New-NsxtLdap -dcMachineName dc-sfo01 -protocol LDAPS -startTtls false -domain sfo.rainpole.io -baseDn "ou=Security Users,dc=sfo,dc=rainpole,dc=io" -bindUser svc-vsphere-ad@sfo.rainpole.io -bindPassword VMw@re1! -certificate Root64.cer
+        This example create an Active Directory Identity Source over LDAPS
+
+        .EXAMPLE
+        New-NsxtLdap -dcMachineName dc-sfo01 -protocol LDAP -startTtls false -domain sfo.rainpole.io -baseDn "ou=Security Users,dc=sfo,dc=rainpole,dc=io" -bindUser svc-vsphere-ad@sfo.rainpole.io -bindPassword VMw@re1!
```

Comment:
```suggestion
        New-NsxtLdap -dcMachineName dc-sfo01 -protocol LDAP -startTtls false -domain sfo.rainpole.io -baseDn "ou=Security Users,dc=sfo,dc=rainpole,dc=io" -bindUser svc-nsx-ad@sfo.rainpole.io -bindPassword VMw@re1!
```

Do you want to use an nsx-to-ad example account?

Reply:
Good point, just didn't have one for testing :)